recipes/hkl/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+test -d m4 || mkdir m4
+# gtkdocize || exit 1
+
+export python=$PYTHON
+export PATH=$PREFIX/bin:$PATH
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig"
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$PKG_CONFIG_LIBDIR
```

Comment:
Why do you need the two lines above?

Reply:
Good question. The first line came in historically from https://github.com/NSLS-II/lightsource2-recipes/blob/f5ba7dbac70a519f3a96720f4e017a7f6f021c93/recipes-tag/hkl/build.sh#L8. I think I had to add the second line during the course of the build to fix some issues (don't remember frankly speaking, was too long ago). I can run the build with the cleaned `build.sh` and see what's going on now.